src/index.ts

```diff
@@ -1,16 +1,16 @@
-type EventType = string | symbol;
+export type EventType = string | symbol;
 
 // An event handler can take an optional event argument
 // and should not return a value
-type Handler = (event?: any) => void;
-type WildcardHandler= (type: EventType, event?: any) => void
+export type Handler = (event?: any) => void;
+export type WildcardHandler= (type: EventType, event?: any) => void
```

Comment:
nit: should the return type here be `any`? Mitt doesn't care about it, but I wonder what this does for `async` handlers that inherently return `Promise<void>`...

Reply:
Oooh good point, I'm not sure. Or maybe `void | Promise<void>` ?

Reply:
I added some type tests and it looks like `void` actually does ignore the return value. I was misinformed!